README.md

```diff
@@ -83,5 +83,5 @@ result = table.search([100, 100]).limit(2).to_pandas()
 ```
 
 ## Blogs, Tutorials & Videos
-* 📈 <a href="https://blog.eto.ai/benchmarking-random-access-in-lance-ed690757a826">2000x better performance with Lance over Parquet</a>
+* 📈 <a href="https://blog.lancedb.com/announcing-lancedb-5cb0deaa46ee-2/">2000x better performance with Lance over Parquet</a>
```

Comment:
Should we actually use this one? https://blog.lancedb.com/announcing-lancedb-5cb0deaa46ee-2/ ?

Reply:
that link is the same as in the diff, no? Or are you considering linking an entirely different page?

Reply:
Thanks @moritzwilksch could you rebase this PR?

It should be good to go

Reply:
the new link also seems dead now